10 Indicator Fixes (alticons)/gamedata/scripts/z_alticons.script

```diff
@@ -0,0 +1,56 @@
+size_adjust = 0.75
+
+function alticons.actor_status.UIIndicators:InitControls()
```

Comment:
I do not know if this is the best way to patch this function, but for the record, UI Rework implements `z_alticons` to monkey patch the `alticons` function. If we also implement here it will overwrite UI Rework's or impliment the monkey patch if they do not have that mod so its a win-win.